torch/csrc/jit/jit_log.cpp

```diff
@@ -26,6 +71,14 @@ void TORCH_API set_jit_logging_levels(std::string level) {
   JitLoggingConfig::getInstance().setLoggingLevels(level);
 }
 
+void TORCH_API set_jit_logging_output_stream(std::ostream& stream) {
+  JitLoggingConfig::getInstance().setOutputStream(stream);
+}
+
+TORCH_API std::ostream& get_jit_logging_output_stream()  {
```

Comment:
I think TORCH_API is only needed at declaration of a function, the definition doesn't need it.

Reply:
I pushed the change again because it was giving windows build errors. It couldnt find the symbol.